intelligence-services/latest/config/comprehend.md

```diff
@@ -524,10 +524,13 @@ For more details on extending the features of Digital Workspace, see the Alfresc
 ### Digital Workspace - Docker service definition
 
 ```yaml
-digital-workspace:
-    image: quay.io/alfresco/alfresco-digital-workspace:1.5.0
+  digital-workspace:
+    image: quay.io/alfresco/alfresco-digital-workspace:2.1.0-adw
+    environment:
+      BASE_PATH: ./
+      APP_CONFIG_PLUGIN_AI_SERVICE: "true"
     volumes:
-      - ./app.extensions.json:/usr/share/nginx/html/assets/app.extensions.json
+      - ./ai-view-extension.json:/usr/share/nginx/html/assets/plugins/ai-view-extension.json
 ```
 
-In the above `docker-compose` snippet, the modified `app.extensions.json` configuration file must be mounted in the Digital Workspace container instead of the standard one (from Content Services deployment).
+In the above `docker-compose` snippet, the modified `ai-view-extension.json` configuration file must be mounted in the Digital Workspace container instead of the standard one (from the Content Services deployment). The environment variable `APP_CONFIG_PLUGIN_AI_SERVICE:` when set to `true` allows the Digital Workspace to generate transcripts for audio and video files, including indexing and metadata generation.
```

Comment:
```suggestion
In the above `docker-compose` snippet, the modified `ai-view-extension.json` configuration file must be mounted in the Digital Workspace container. The environment variable `APP_CONFIG_PLUGIN_AI_SERVICE:` when set to `true` allows the Digital Workspace to generate transcripts for audio and video files, including indexing and metadata generation.
```
In this new version we are not replacing the original, we are just adding this file